pkg/kv/kvserver/raftlog/encoding.go

```diff
@@ -23,41 +23,41 @@ import (
 type EntryEncoding byte
 
 const (
-	// RaftVersionStandard is the default encoding for a CockroachDB raft log
+	// EntryEncodingStandard is the default encoding for a CockroachDB raft log
 	// entry.
 	//
 	// This is a raftpb.Entry of type EntryNormal whose Data slice is either empty
-	// or whose first byte matches RaftVersionStandardPrefixByte. The subsequent
+	// or whose first byte matches EntryEncodingStandardPrefixByte. The subsequent
 	// eight bytes represent a CmdIDKey. The remaining bytes represent a
 	// kvserverpb.RaftCommand.
-	RaftVersionStandard EntryEncoding = 0
-	// RaftVersionSideloaded indicates a proposal representing the result of a
+	EntryEncodingStandard EntryEncoding = 0
```

Comment:
Not in this PR, but I think these enum values can be made automatic, starting with `iota`. This would further rule out doubts of the reader on whether these can be used on the wire, and why the values jump to 255.